src/plugins/intel_gpu/src/runtime/debug_configuration.cpp

```diff
@@ -519,4 +522,16 @@ bool debug_configuration::is_target_iteration(int64_t iteration) const {
     return false;
 #endif
 }
+
+bool debug_configuration::get_env(std::string key, int &val) const {
```

Comment:
Is this to apply to master?

Reply:
I also once thought of using Gemm environment variables into debug_config list directly. However, I did this because it  would be two problems.
1. It can be too many debug_config list due to many environment variables that apply only to specific kernels.
2. It can be unexpected confusion caused by adding many environment variables (We can't limit environment variable name). So, I added env_var a variable that once again requires the user's clear intent.

However, I also agree many environment variables that are not disclosed to debug_config may cause difficulties later. Does exposing environment variable directly to the debug_config list seem better?

Reply:
I think it is OK to have a new single debug control like "GEMM_TILE_SIZE=m,n,k" or having internal(unexposed) env variable.

But even with current code style, the problem is that... `OV_GPU_EnvVar` is actually not needed.  The code below would work just same.
```
    GPU_DEBUG_IF(1) {
        int val;
        if (debug_config->get_env("GEMM_TILE_M", val))
            tuning_data.tile_m_size = val;
        if (debug_config->get_env("GEMM_TILE_N", val))
            tuning_data.tile_n_size = val;
        if (debug_config->get_env("GEMM_TILE_K", val))
            tuning_data.tile_k_size = val;
        if (debug_config->get_env("GEMM_TILE_SIMD", val))
            tuning_data.simd_size = val;
    }
```

Reply:
Frankly, control tile size is useful in unit-test level, but it's not in model level. I don't have nice idea, Remove it.